src/serving.ts

```diff
@@ -291,4 +302,37 @@ export class Server {
       resolve();
     });
   }
+
+  // Issues a request to reCAPTCHA to verify the request is from a human. Note
+  // that this executes only if a valid config is provided.
+  verifyRecaptcha() {
+    return (req: express.Request, res: express.Response, next: express.NextFunction) => {
+      const recaptchaConfig = this.config.recaptchaConfig;
+
+      // Skip reCAPTCHA verification if the requisite config was not provided.
+      if (!recaptchaConfig) {
+        return next();
+      }
+
+      axios.default
+        .post('https://www.google.com/recaptcha/api/siteverify', {}, {
+          params: {
+            secret: recaptchaConfig.secretKey,
+            response: req.body.recaptchaToken,
+          }
+        })
+        .then((response: axios.AxiosResponse) => {
+          if (response.data[errorAccessor]) {
+            res.status(UNAUTHORIZED).send('reCAPTCHA request failed: ' + response.data[errorAccessor]);
+          } else if (!response.data.success
+            || Number(response.data.score) < Number(recaptchaConfig.threshold)
```

Comment:
Nit: Prefer indentation of these lines to match with the left parentheses of the else if above